docs/operator-docs/update-cumulus-dependencies.md

```diff
@@ -62,3 +62,20 @@ Note that there may be configuration changes required with a new version. Please
 When breaking changes have been introduced, the Cumulus Core team will publish a document with instructions on migrating from one version to another. *IMPORTANT:* When upgrading through many versions, each migration should be done in the order of release (if going from version 1.z.0 from 1.x.0, 1.x.0 -> 1.y.0 -> 1.z.0).
 
 The migration documents published on release can be found [here](https://nasa.github.io/cumulus/docs/upgrade/upgrade-readme).
+
+## Maintaining the current Version of a Dependency
+
+If you wish to maintain the current version for a specific dependency, make sure to save that version to the package.json with this command.
+
+```shell
+npm install [<@scope>/]<name>@<version> --save --save-exact
```

Comment:
Using an actual example might be a bit more clear?

`npm install @cumulus/deployment@1.11.3 --save --save-exact`

Also, I think `--save-exact` might be redundant when you are already specifying an exact version (no `~` or `^`), but I could be wrong

Reply:
I would need to use the --save-exact If I'm taking out the section about editing the package.json directly.